libraries/aws_vpn_connection.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsVPNConnection < AwsResourceBase
+  name 'aws_vpn_connection'
+  desc 'Verifies settings for a single AWS VPN Connection.'
+
+  example "
+   describe aws_vpn_connection(vpn_connection_id: 'vc-014aef8a0689b8f43') do
+     it { should exist }
+   end
+
+   describe aws_vpn_connection('vc-014aef8a0689b8f43') do
+     it { should exist }
+   end
+  "
+
+  def initialize(opts = {})
+    opts = { vpn_connection_id: opts } if opts.is_a?(String)
+    super
+    validate_parameters(required: [:vpn_connection_id])
+    validate_identifier
+    fetch
+  end
+
+  def exists?
+    !failed_resource? && !!@response || !@response.empty?
+  end
+
+  def to_s
+    "VPN connection ID: #{@opts[:vpn_connection_id]}"
+  end
+
+  def resource_id
+    @opts ? @opts[:vpn_connection_id] : ''
```

Comment:
We should not pass an empty string. If we pass an empty string then the resource_id will be empty. 

We can @response[: vpn_connection_id]